app/src/validators/enquiry.ts

```diff
@@ -33,6 +33,7 @@ const schema = {
     body: Joi.object({
       enquiryId: Joi.string().required(),
       activityId: Joi.string().required(),
+      atsClientNumber: Joi.string().allow(null).max(255),
```

Comment:
SQL Text types do not have an upper bound having a max of 255 isn't really needed unless there's another kind of constraint in the business rules.

Reply:
This is consistent with the rest of the base. Limit via the API as we're using `text` in the db.